build/Signing.cs

```diff
@@ -80,20 +96,40 @@ static void SignWithSignTool(AbsolutePath[] files)
     
     static void SignWithAzureSignTool(AbsolutePath[] files)
     {
-        var arguments = "sign " +
-            $"--azure-key-vault-url \"{Build.AzureKeyVaultUrl}\" " +
-            $"--azure-key-vault-client-id \"{Build.AzureKeyVaultAppId}\" " +
-            $"--azure-key-vault-client-secret \"{Build.AzureKeyVaultAppSecret}\" " +
-            $"--azure-key-vault-certificate \"{Build.AzureKeyVaultCertificateName}\" " +
-            "--file-digest sha256 ";
-
-        foreach (var file in files)
+        
+        var lastException = default(Exception);
+        foreach (var timestampUrl in SigningTimestampUrls)
         {
-            arguments += $"\"{file}\" ";
-        }
+            Logging.InBlock($"Trying to time stamp using {timestampUrl}", () =>
+            {
+                try
+                {
+                    var arguments = "sign " +
+                        $"--azure-key-vault-url \"{Build.AzureKeyVaultUrl}\" " +
+                        $"--azure-key-vault-client-id \"{Build.AzureKeyVaultAppId}\" " +
+                        $"--azure-key-vault-client-secret \"{Build.AzureKeyVaultAppSecret}\" " +
+                        $"--azure-key-vault-certificate \"{Build.AzureKeyVaultCertificateName}\" " +
+                        "--file-digest sha256 " +
+                        $"--timestamp-rfc3161 \"{timestampUrl}\" ";
```

Comment:
Did we lose that in the nuke conversion, or previously?
Do we need to pull the cord-that-shall-not-be-named for a certain range?

Reply:
We lost it in the conversion - what happened previously was it'd sign with azure sign tools, then timestamp using signtools - i've changed it to timestamp at the same time.